experimental/dds/tree2/src/simple-tree/proxies.ts

```diff
@@ -133,13 +133,14 @@ export function getOrCreateNodeProxy<TSchema extends TreeNodeSchema>(
 	} else {
 		// Fallback to createNodeProxy if needed.
 		// TODO: maybe remove this fallback and error once migration to class based schema is done.
-		output = createNodeProxy<TSchema>(flexNode);
+		output = createNodeProxy<TSchema>(flexNode, false);
 	}
 	return output;
 }
 
```

Comment:
I know these weren't documented before, but could we add param docs for `allowAdditionalProperties` to these functions since we're adding it?

Reply:
Done. Also document customTargetObject.